rock.osdeps

```diff
@@ -34,7 +34,10 @@ binding_of_caller:
     gem: binding_of_caller
 
 activesupport:
-    gem: activesupport <= 4.2.5.1
+  ubuntu: ruby-activesupport
+  debian: ruby-activesupport
+  default:
+    gem: activesupport
```

Comment:
That's right, but the grape gem also has activesupport in its dependency list (unversioned), on new bootstraps on the buildserver it happens that the grape gem is evaluated fist and tries to install the current version of activesupport via the gem dependency mechanism, adding it as osdep it makes sure that activesupport is already installed, when the grape gems dependencies are evaluated.

Or is there a way to tell autoproj to install specific gems first?